luthier/config/src/dimensions.lua

```diff
@@ -121,6 +122,15 @@ local searchProviders = dimensionUtils.searchProviders
 local keyValueStores = dimensionUtils.keyValueStores
 
 return {
+    testDimension = {
+        longName = "a longName for testing",
+        description = "a description for testing",
+        fields = FIELDSETS.test,
+        category = "a category for testing",
+        type = "TestDimensionFactory",  -- type
```

Comment:
Would remove the `--type` comment at the end of the line. It's redundant.

Reply:
Fixed.